### PR TITLE
Add enhavo/app-bundle recipe

### PR DESCRIPTION
**Pull request by gseidel**
| Q             | A
| ------------- | ---
| License       | MIT

Adding a recipe for the enhavo/app-bundle

### CODE REVIEW

**Comment by gseidel** [comment]
@Nyholm Maybe you can make a review, because you also reviewed my PR before #915

File: enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+imports:
+    -
+        path: '@enhavo/app/services/*'
```
Comment (Nyholm) [inline enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml:3]: You dont need this for a bundle. The bundle will register their own services. See https://github.com/enhavo/EnhavoAppBundle/blob/v0.9.2/DependencyInjection/EnhavoAppExtension.php#L46-L61

File: enhavo/app-bundle/0.9/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "Enhavo\\Bundle\\AppBundle\\EnhavoAppBundle": ["all"]
+    },
+    "container": {
+        "locale": "en"
+    },
+    "env": {
+        "MAILER_FROM": "no-reply@email.com",
+        "MAILER_TO": "my@email.com",
+        "MAILER_NAME": "enhavo",
+        "MAILER_DELIVERY_ADDRESS": "my@email.com"
```
Comment (Nyholm) [inline enhavo/app-bundle/0.9/manifest.json:12]: These variables are never used. You should make sure to add those to `enhavo_app.yaml`

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. It looks good. Just some minor things

File: enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml (hunk 1 above)
Comment (gseidel) [reply enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml:3]: This isn't a configuration file for `Symfony`. It looks similar, but it is used for a webpack plugin within enhavo. (This is how I avoid using Typescript files in the recipe)

https://github.com/enhavo/dependency-injection#import

Comment (Nyholm) [reply enhavo/app-bundle/0.9/assets/services/enhavo/enhavo_app.yaml:3]: Ah, I see. Okey.

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**